src/components/Util/Loading.tsx

```diff
@@ -23,6 +23,7 @@ const Loading: React.FC = () => (
         size="xl"
       />
       <Text>Loading...</Text>
+      <Text>Cargando...</Text>
```

Comment:
Nit (based on opinion): Put English Loading above the circle and the Cargando script below the circle